admin-dev/themes/new-theme/js/components/form/currency-symbol-updater.ts

```diff
@@ -52,11 +52,13 @@ export default class CurrencySymbolUpdater {
   }
 
   private getSymbol(select: HTMLSelectElement): string {
+    const defaultCurrencySymbol: string = select.dataset.defaultCurrencySymbol || '';
```

Comment:
It works in old vanilla JS with undefined value at least But there may be some issues in case it contains a boolean value yes Your suggestion us much better